automated/linux/armnn/armnn.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# shellcheck disable=SC1091
+. ../../lib/sh-test-lib
+OUTPUT="$(pwd)/output"
+OUTPUT_FILE="${OUTPUT}/results.txt"
+PYTEST="false"
+ARMNN_TARBALL="https://snapshots.linaro.org/components/armnn/latest/armnn-full.tar.xz"
```

Comment:
Do you want to detect "detect_abi" if the target device is other than "armeabi" and "arm64" and then skip the test ?

Reply:
I don't think this is necessary. These tests can be run on other devices as long as a device specific ARMNN_TARBALL is given as a parameter.